script/Ciel/Day 14/14_01B_03_CIEL14B_6.txt

```diff
@@ -662,37 +721,38 @@ As I squint my eyes from the pain, my barely alive consciousness cuts away.
 [sha:c9583fa5e1cc81f9c281b01a5af9fd36c93236ee]{
 -- Page 332, Offset 41216.
 -- 先輩、と呼びかけた声は、音にならなかった。
-The sound of Senpai's voice calling out to me does not reach my ears.
+I try to call Senpai's name, but no sound comes out.
 }
 [sha:3550cdbd556e3b6c6f7dcb41a11432ad611d31c2]{
 -- Page 333, Offset 41217.
 -- 「ひどい―――何があったんですか！？
-"How awful―――What on earth happened out there?!
+"This is awful――what on earth happened out there?!
 }
 [sha:c6d8e4b7ba013d5b3e1319e387684d180522b59f]{
 -- Page 333, Offset 41218.
 -- しっかり、しっかりしてください、遠野くん……！」
-Get... Get a hold of yourself, Tohno-kun...!"
+Hold on... Stay with me, Tohno-kun!"
 }
 [sha:aff65ae45aacf478d92271821f06eca149a93ddb]{
 -- Page 333, Offset 41219.
 -- 何って、まあ、いろいろあったんだけど―――
-You're asking what happened? Well a lot of things, but―――
+You're asking what happened? Well, a lot of things, but―――
 }
 [sha:9138e83cdfb4cba7704da425ae27de9c4e66f20f]{
 -- Page 334, Offset 41220.
 -- 『大丈夫。もう何も心配はないですよ』
-'It's okay. There's nothing to be worried about.'
+'It's okay. There's nothing to worry about anymore.'
 }
 [sha:19fae3cee1777ababbef60ebede6f65b9fe8e08d]{
 -- Page 334, Offset 41221.
 -- そう口にしようとして、喉はおろか、呼吸さえできない事に、気がついた。
-I try to speak out those words, but then realized that I couldn't speak, let alone breathe.
+I try to reassure her, only to find that I can't even breathe, let alone speak.
 }
 [sha:907e4919c111cd38460755eabe56bcf0c3d46988]{
 -- Page 335, Offset 41222.
 -- 「……！　……！」
-"...! ...!"
+// She says Tohno-kun here. I know it's a style choice to elide, but no ...!
+"Tohno-kun! Tohno-kun!"
```

Comment:
Could use those weird blocks they sometimes use to censor words. Unlike the Arcueid mumblings in the file earlier, people probably don't need a TL for a name they've heard a millions times at this point, so we can keep the flavor.